src/pages/patterns/tables/basic-formatting.mdx

```diff
@@ -0,0 +1,447 @@
+
+![Date and Time](figma:patterns-tables-basic-formatting-date-time.png)
+
+## States
+
+### Rows and cells
+
+![Row and Cell States](figma:patterns-tables-basic-formatting-row-cell-states.png)
+
+<table>
+  <thead>
+    <tr>
+      <th width="30%">Row/Cell state</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>1. Empty cell</td>
+      <td>Cells with no values should be left blank. Do not use a hyphen, en, or em dash.</td>
+    </tr>
+    <tr>
+      <td>2. Hover</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>3. Focused</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>4. Selected</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>5. Focused and selected</td>
+      <td>
+        The last row checked is the focused row, unless the user clicks on a different row or uses
+        the keyboard to change focus.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+### Loading
+
+Skeleton screens are used to indicate that a screen is loading. They are perceived as being shorter in duration when
+compared against a blank screen or a spinner.
+
+- Implement the [Skeleton](/components/skeleton) component in each cell up to three rows
+- For a table with three or more rows, show the first three rows as skeletons with the header cells being shown at all
+  times to provide instant context of incoming tabular data
+- The width of the Skeleton should match the width of the intended cell content, while the Skeleton’s height will be 60%
+  of the font size
+
+import TableLoading from '../../../examples/table/TableLoading.tsx';
+import TableLoadingCode from '!!raw-loader!../../../examples/table/TableLoading.tsx';
+
+<CodeExample code={TableLoadingCode}>
+  <TableLoading />
+</CodeExample>
+
+## Localization & internationalization
+
+Our tables have been designed to accommodate localization and internationalization standards so they do not break the
+experience.
+
+<ImageFigure
+  caption="Bulgarian example"
+  imgAlt="Bulgarian example"
+  imageSource={props.data.tablesLocalizationBulgarian.childFile.childImageSharp.gatsbyImageData}
+/>
+
+<ImageFigure
+  caption="Arabic RTL example"
+  imgAlt="Arabic RTL example"
+  imageSource={props.data.tablesLocalizationArabic.childFile.childImageSharp.gatsbyImageData}
+/>
+
+## Accessibility
+
+### General considerations
+
+All Garden [Tables](/components/table) are built with accessible markup.
+
+### Screen reader order
+
+People using screen readers can have the row and column cells read aloud as they navigate through the table. Screen
+readers speak one cell at a time and reference the associated header cells, so the reader doesn’t lose context.
+
+### Additional resources
+
+- [https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Advanced](https://developer.mozilla.org/en-US/docs/Learn/HTML/Tables/Advanced)
+
+---
+
+<h2>Feedback</h2>
+
+Help improve this page. [Give us your feedback](https://forms.gle/VFD799SvNfs2yiFC7)
```

Comment:
We'll either need to remove and do a (quick) follow-on to add this back, or wait for my doc review.